pymongo/bulk.py

```diff
@@ -408,6 +408,7 @@ def execute_command(
         generator: Iterator[Any],
         write_concern: WriteConcern,
         session: Optional[ClientSession],
+        operation: Optional[str] = "TEST_OPERATION",
```

Comment:
Can we just have this be `_Op.TEST`?

Additionally, for typing, is there a case where we would pass `operation=None` because if not, we could also type this to just be of type `str`.

Reply:
Found a case reading through, so ignore the second part :)

Reply:
Good catch. Agree we should remove "TEST_OPERATION" and make this a required arg.

Reply:
I've made `operation` required everywhere it appears for consistency with the spec and for ease of future refactors/development.